From 59a1759daebd34b11055e122630765904b11c407 Mon Sep 17 00:00:00 2001
From: Joachim Wiberg <troglobit@gmail.com>
Date: Sun, 10 Aug 2025 18:52:54 +0200
Subject: [PATCH 28/30] arm64: dts: mediatek: mt7986a: rename BPi R3 ports to
 match case
Organization: Wires

For ref. see: https://wiki.banana-pi.org/File:Bpi-r3_Metal_case.jpg

Also, comment out gpio-keys (reset and wps) since they clash with with
the PCIe controller.

Signed-off-by: Joachim Wiberg <troglobit@gmail.com>
---
 .../boot/dts/mediatek/mt7986a-bananapi-bpi-r3.dts  | 14 +++++++-------
 1 file changed, 7 insertions(+), 7 deletions(-)

diff --git a/arch/arm64/boot/dts/mediatek/mt7986a-bananapi-bpi-r3.dts b/arch/arm64/boot/dts/mediatek/mt7986a-bananapi-bpi-r3.dts
index ed79ad1ae871..1c7c666e3173 100644
--- a/arch/arm64/boot/dts/mediatek/mt7986a-bananapi-bpi-r3.dts
+++ b/arch/arm64/boot/dts/mediatek/mt7986a-bananapi-bpi-r3.dts
@@ -46,7 +46,7 @@ fan: pwm-fan {
 		pwms = <&pwm 0 10000>;
 		status = "okay";
 	};
-
+/*
 	gpio-keys {
 		compatible = "gpio-keys";
 
@@ -62,7 +62,7 @@ wps-key {
 			gpios = <&pio 10 GPIO_ACTIVE_LOW>;
 		};
 	};
-
+*/
 	/* i2c of the left SFP cage (wan) */
 	i2c_sfp1: i2c-gpio-0 {
 		compatible = "i2c-gpio";
@@ -418,27 +418,27 @@ port@0 {
 
 		port@1 {
 			reg = <1>;
-			label = "lan0";
+			label = "lan1";
 		};
 
 		port@2 {
 			reg = <2>;
-			label = "lan1";
+			label = "lan2";
 		};
 
 		port@3 {
 			reg = <3>;
-			label = "lan2";
+			label = "lan3";
 		};
 
 		port@4 {
 			reg = <4>;
-			label = "lan3";
+			label = "lan4";
 		};
 
 		port5: port@5 {
 			reg = <5>;
-			label = "lan4";
+			label = "sfp2";
 			phy-mode = "2500base-x";
 			sfp = <&sfp2>;
 			managed = "in-band-status";
